### PR TITLE
Add new model definitions for watsonx testing

### PR DESCRIPTION
**Pull request by npalaska**

### CODE REVIEW

File: projects/kserve/testing/models/watsonx_base.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  kserve:
+    resource_request:
+      cpu: 8
+    extra_env:
+      TRANSFORMERS_CACHE: "/tmp/transformers_cache"
+inference_service:
+  model_format: "pytorch"
+  min_replicas: 1
+  maxReplicas: 1
+  single_container: true
```
Comment (kpouget) [inline projects/kserve/testing/models/watsonx_base.yaml:11]: can you move this line to the beginning of `serving_runtime`?
```
serving_runtime:
  single_container: true
  kserve:
```
(it's a ServingRuntime properly, not ISVC, I don't know how I got that wrong in my first try 🤦🏻 )

Could you also rename this file to `base_....yaml`? I'm not sure what base name would accurately describe this file, maybe `tgis-only`? 

and `maxReplicas: 1` property does not exist 🙃 (please remove it from here)

Comment (npalaska) [reply projects/kserve/testing/models/watsonx_base.yaml]: Fixed.

File: projects/kserve/testing/models/base-standalone-tgis.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+serving_runtime:
+  single_container: true
```
Comment (kpouget) [inline projects/kserve/testing/models/base-standalone-tgis.yaml:2]:
```suggestion
  container_flavor: tgis
```

**Comment by kpouget** [comment]
thanks @npalaska, merging.

**Pull request merged**